osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/ImportHighScoresCommand.cs

```diff
@@ -116,7 +116,7 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
                         await checkSlaveLatency(db, cancellationToken);
 
                     Console.WriteLine($"Fetching next scores from {lastProcessedId}...");
-                    var highScores = await db.QueryAsync<HighScore>($"SELECT h.*, s.id as new_id FROM {highScoreTable} h "
+                    var highScores = await db.QueryAsync<HighScore>($"SELECT h.*, s.id as new_id, s.user_id as new_user_id FROM {highScoreTable} h "
```

Comment:
Why is this added to the import command, and why isn't it added to the _watch_ command?

I didn't question this initially, however I noticed some logging looked wonky that I was gonna go and fix (importer not logging deletion numbers), and then I realised that it probably doesn't matter because import is _one-shot_ and what this probably primarily wants to be fixing is the _watcher_, no?

Reply:
yes. too many commands.